fall23/deepface/face_matching.ipynb

```diff
@@ -0,0 +1,3368 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #88.    fr.detect_and_match_faces()
</pre><p>Would be good to get a better sense of how this is working.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-atfal-mafkoda-missing-children/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>